include/runtime/stackmgr.h

```diff
@@ -35,7 +37,29 @@ class StackManager {
     uint32_t VPos;
   };
 
-  using Value = ValVariant;
+  struct Handler {
+    Handler(AST::InstrView::iterator E, uint32_t V, uint32_t F, uint32_t H,
+            uint32_t C)
+        : CatchCaluse(), EndIt(E), VPos(V), FPos(F), HPos(H), CPos(C) {}
+    // nullptr stands for catch all clause
+    std::vector<
+        std::pair<Runtime::Instance::TagInstance *, AST::InstrView::iterator>>
```

Comment:
I'm not sure whether it's a proper way to distinguish the tag by the address. The current implementation may compare two tag address to indicate whether they are the same tag.